code/modules/admin/topic.dm

```diff
@@ -695,7 +695,7 @@
 		dat += "</tr></table>"
 
 	//BoS (Steel Blue)
-		dat += "<table cellpadding='1' cellspacing='0' width='100%'>"
+	/*	dat += "<table cellpadding='1' cellspacing='0' width='100%'>"
```

Comment:
No need to comment this out.
Better to have Legacy code that isn't used than need it and have to call the right guy to re-enable it